Tests/FluentAssertions.Specs/Equivalency/BasicEquivalencySpecs.cs

```diff
@@ -2497,7 +2497,7 @@ public void When_the_compile_time_type_does_not_match_the_equality_comparer_type
                 opt.Using<ConcreteClass, ConcreteClassEqualityComparer>());
 
             // Assert
-            act.Should().NotThrow();
+            act.Should().Throw<InvalidOperationException>().WithMessage("No members were found for comparison*");
```

Comment:
What changed here?

Reply:
I'm not 100% sure, but somehow, the comparer wasn't used until my changes.